commit 75b45f95766e3e555ab2f040f67295a3cd00f5a4
Author: onettboots <blackcocopet@gmail.com>
Date:   Fri Jun 23 14:00:07 2023 +0000

    [REAPPLY] kernelsu: we're non GKI and non KPROBES build
            This is an automated commit using script from
            https://github.com/realme-sm7125/myscripts/blob/main/kernelsu.sh
    
    Signed-off-by: Mayur <ultramayur123@gmail.com>

diff --git a/drivers/kernelsu/ksu.c b/drivers/kernelsu/ksu.c
index c2da89022624..1dc3901ba932 100644
--- a/drivers/kernelsu/ksu.c
+++ b/drivers/kernelsu/ksu.c
@@ -53,12 +53,12 @@ int __init kernelsu_init(void)
 
 	ksu_uid_observer_init();
 
-#ifdef CONFIG_KPROBES
+//#ifdef CONFIG_KPROBES
 	ksu_enable_sucompat();
 	ksu_enable_ksud();
-#else
-	pr_alert("KPROBES is disabled, KernelSU may not work, please check https://kernelsu.org/guide/how-to-integrate-for-non-gki.html");
-#endif
+//#else
+//	pr_alert("KPROBES is disabled, KernelSU may not work, please check https://kernelsu.org/guide/how-to-integrate-for-non-gki.html");
+//#endif
 
 	return 0;
 }
diff --git a/drivers/kernelsu/ksu.h b/drivers/kernelsu/ksu.h
index cdffb5aece87..fcaa393f22c7 100644
--- a/drivers/kernelsu/ksu.h
+++ b/drivers/kernelsu/ksu.h
@@ -4,7 +4,7 @@
 #include "linux/types.h"
 #include "linux/workqueue.h"
 
-#define KERNEL_SU_VERSION KSU_VERSION
+#define KERNEL_SU_VERSION (10818)
 #define KERNEL_SU_OPTION 0xDEADBEEF
 
 #define CMD_GRANT_ROOT 0
